.github/renovate.json

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "rebaseWhen": "behind-base-branch",
+    "labels": [
+        "dependencies"
+    ],
+    "extends": [
+        "config:recommended"
+    ],
+    "packageRules": [
+        {
+            "matchPackageNames": [
+                "composer/composer"
+            ],
+            "enabled": true
+        },
+        {
+            "matchFileNames": [
+                "lts/Dockerfile"
+            ],
+            "matchPackageNames": [
+                "composer/composer"
+            ],
+            "versioning": "semver",
+            "commitMessageSuffix": " (LTS)",
+            "separateMinorPatch": true,
+            "additionalBranchPrefix": "lts-"
+        },
+        {
+            "matchFileNames": [
+                "latest/Dockerfile"
+            ],
+            "matchPackageNames": [
+                "composer/composer"
+            ],
+            "commitMessageSuffix": " (Latest)",
+            "separateMinorPatch": true,
+            "additionalBranchPrefix": "latest-"
+        }
+    ],
+    "customManagers": [
+        {
+            "customType": "regex",
+            "managerFilePatterns": [
+                "/latest/Dockerfile$/"
+            ],
+            "matchStrings": [
+                "ENV COMPOSER_VERSION=[\"']?(?<currentValue>\\d+\\.\\d+\\.\\d+)[\"']?\\s+"
+            ],
+            "datasourceTemplate": "github-releases",
+            "depNameTemplate": "composer/composer"
+        },
+        {
+            "customType": "regex",
+            "managerFilePatterns": [
+                "/lts/Dockerfile$/"
+            ],
+            "matchStrings": [
+                "ENV COMPOSER_VERSION=[\"']?(?<currentValue>2\\.\\d+\\.\\d+)[\"']?\\s+"
```

Comment:
Should this not be 2.2.\d+

Not sure why/how this matches lts releases right now. Also note we'll most likely have a second lts branch soon like 2.9 + 2.2 + latest, not sure if this matters yet.

Reply:
Wait, 2 LTS branches? That kind of does mess up the current directory structure format; but we can change that of course.

As for the pattern matching, I do think it matters.

Reply:
I am not sure which of those the generic `composer:lts` tag should target then though 😅

Reply:
Well not sure what i do with the old lts might drop it too but would be good to make tags more specific anyway to avoid confusion imo. Lts is too generic. 2.2-lts would be nicer.

Reply:
Happy to help rework it based on the changes you folks want to make.  So would changing it to be `2.2.\d+` make sense now or after the directory changes.

Reply:
IMO `composer:lts` should be deprecated as it is meaningless. Keep it pointing to 2.2-lts for BC then kill it when 2.2 dies.

Other than that sounds good to me but I am only vaguely aware of what happens in this repo so I'll let @alcohol handle the rest :D

Reply:
@inverse I would rather drop the `lts` tag then, and use both `2.2-lts` and `2.9-lts`. Or even better perhaps, just drop the `-lts` suffix in general. If getcomposer.org documents which versions or branches are LTS, we can just keep using the branch aliases that already exist. That would be `2.2` and `2.9` in this case.

Reply:
@alcohol Updated the config to the new repo structure

Will reference the 2.2/dockerfile ones with 2.2
Will reference latest/Dockerfile ones with Latest

No lts mentions anymore